frontend/src/pages/WhatIsHealthEquity/WhatIsHealthEquityPage.module.scss

```diff
@@ -385,58 +438,3 @@
     text-decoration: underline !important;
   }
 }
-
-/** ------------- FAQ ------------- **/
-
-.FaqGrid {
```

Comment:
I don't think you actually changed any of these FAQ styles, right? What's the reason for moving them?

Reply:
For some reason there were two FAQ sections in the previous version of this file, I consolidated them back into one.